test/fixtures/test/action-events.json

```diff
@@ -294,7 +294,7 @@
           "type": "clinicians"
         }
       },
-      "form": {
+      "form_revision": {
         "data": {
           "id": "11111",
           "type": "forms"
```

Comment:
Form revisions have `type: "form-revisions"`.

Reply:
didn't realize there was a new type, but I guess that makes sense.